openff/toolkit/tests/test_toolkits.py

```diff
@@ -2834,7 +2836,31 @@ def test_assign_partial_charges_am1bcc(self):
         for pc in molecule._partial_charges:
             charge_sum += pc
             abs_charge_sum += abs(pc)
-        assert abs(charge_sum) < 0.001 * unit.elementary_charge
+        assert abs(charge_sum) < 1e-8 * unit.elementary_charge
```

Comment:
Not blocking: I wonder if this molecule (acetate) should have a smaller error than 1e-8?

Reply:
I've tightened all instances of this down to 1e-10, though if we want to be really strict, full python float precision touches 1e-15. I think 1e-10 is fine for now, though, since too tight a tolerance will win us lots of false negatives in the tests.